GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using CommandLine;

```

Comment:
It crashes on the Mac. Paths.GetServiceDataRoot() returns ```/usr/share/GVFS``` to which user may have no write access.

```
Unhandled Exception: GVFS.Common.FileBasedCollectionException: Access to the path '/usr/share/GVFS' is denied. ---> System.UnauthorizedAccessException: Access to the path '/usr/share/GVFS' is denied
```

Need to figure out the equivalent of ```C:\ProgramData\GVFS``` directory on Mac. ```/usr/local/share/GVFS``` looks SIP safe. Thoughts?

Reply:
Or just block the command for now, since it's only needed for upgrade, which is also blocked. We need to come back around and fix this up for Mac, but that's out of scope here.

Reply:
added blocking on ```GVFSPlatform.Instance.IsUnderConstruction```.